app/Models/PollerGroup.php

```diff
@@ -43,7 +43,7 @@ public static function boot()
         static::deleting(function (PollerGroup $pollergroup) {
             // handle device pollergroup fallback to default poller
             $default_poller_id = \LibreNMS\Config::get('distributed_poller_group');
-            $pollergroup->devices()->update(['poller_group' => $default_poller_id]);
+            $pollergroup->devices->update(['poller_group' => $default_poller_id]);
```

Comment:
This is probably fine, but could be very slow.

Reply:
The reason is stated in f09a7671703fc9090f439eb13c1d8c538184953e, what do you say?

Reply:
I say we don't need those events right now.  I would rather skip them.

Reply:
Fixed